question/tests/behat/columnsortorder.feature

```diff
@@ -0,0 +1,47 @@
+@core @core_question @javascript
+Feature: An plugin column can be reordered and displayed in the question bank view.
+  In order to reorganise the question bank view columns
+  As an admin
+  I need to be able to reorder them
+
+  Background:
+    Given the following "users" exist:
+      | username | firstname | lastname | email |
+      | teacher1 | Teacher | 1 | teacher1@example.com |
+    And the following "courses" exist:
+      | fullname | shortname | format |
+      | Course 1 | C1 | weeks |
+    And the following "course enrolments" exist:
+      | user     | course | role           |
+      | teacher1  | C1     | editingteacher |
+    And the following "question categories" exist:
+      | contextlevel | reference | name           |
+      | Course       | C1        | Test questions |
+    And the following "questions" exist:
+      | questioncategory | qtype | name                        | user  | questiontext                  | idnumber  |
+      | Test questions   | essay | Test question to be seen | teacher1 | Write about whatever you want | numidnum</1 |
```

Comment:
can you please replace` numidnum</1` to something like idnumber1

Reply:
replaced by `idnumber1`